### ISSUE TITLE
File typ as trigger for filelink

### ISSUE THREAD

**Issue by brtptrs** [nextcloud/nextcloud-filelink#7]
It would be great to have the option to attach all files from certain types. Something like: Send all MS Office Documents as Download link.

**Comment by oparoz** [comment]
I agree. Unfortunately, I think the Filelink API doesn't support that.
Are you aware of an extension which does something like this?
